test/cpp/qps/client_callback.cc

```diff
@@ -289,7 +293,10 @@ class CallbackStreamingPingPongReactor final
       gpr_timespec next_issue_time = client_->NextRPCIssueTime();
       // Start an alarm callback to run the internal callback after
       // next_issue_time
-      ctx_->alarm_.experimental().Set(next_issue_time, [this](bool ok) {
+      if (ctx_->alarm_ == nullptr) {
```

Comment:
In this one I think we don't need the nullptr check because we've already created this RPC by now and are considering to continue to the next operation.

Reply:
Done.